### DISCUSSION TITLE
include current field value in dependsOn Patch

### DISCUSSION THREAD

**Discussion by ianrobertsFF** [laravel/nova-issues#4302] [Ideas & Feature Requests] [answered]
It would be handy if the current value of the field depending on (not the field being depended on) could be sent with `formData` along with the `watchedFields` data.

Handy in general if you need logic that depends on knowing the value of the field you're trying to update, but essential if you're trying to update the value in a KeyValue field

**Comment by crynobone** [comment, marked as answer]
Implemented in 4.19.0 

```php
Text::make('Email')
  ->dependsOn('password', function ($field, $request, $formData) {
    // access current field `$formData->email`
  }),
